packages/transit/src/travel-mode-selector.tsx

```diff
@@ -106,7 +114,10 @@ export default class TravelModeSelector extends React.Component<Props, {}> {
           </>
         ) : (
           <div className="compare-button-wrapper">
-            <div className="compare-button" onClick={onCompare} />
+            <div className="compare-button" id="compare-button" onClick={onCompare} />
+            <div for="compare-button" className="mdl-tooltip mdl-tooltip--right hide-on-mobile">
```

Comment:
I think you want [`htmlFor`](https://reactjs.org/docs/dom-elements.html#htmlfor).